vllm/model_executor/layers/quantization/fp8.py

```diff
@@ -37,6 +37,21 @@
 
 logger = init_logger(__name__)
 
+allow_deep_gemm = False
+if envs.VLLM_USE_DEEP_GEMM:
+    try:
+        import deep_gemm as dg
+        logger.info("Using DeepGemm for fused MoE.")
+        allow_deep_gemm = True
+    except ImportError:
+        logger.warning("Failed to import DeepGemm kernels.")
```

Comment:
It would be nice to additionally check:
```
current_platform.is_cuda() and current_platform.has_device_capability(90)
````
and return false and log a warning otherwise